.gitignore

```diff
@@ -268,37 +268,10 @@ cython_debug/
 # Orga
```

Comment:
refactoring the inputs/outputs a bit allows us to keep the `.gitignore` file clean/maintainable